src/applications/hca/actions.js

```diff
@@ -98,10 +118,55 @@ export function getEnrollmentStatus(formData) {
       ) {
         callFake404(dispatch);
       } else {
-        callFakeSuccess(dispatch, HCA_ENROLLMENT_STATUSES.canceledDeclined);
+        callFakeSuccess(dispatch, HCA_ENROLLMENT_STATUSES.enrolled);
       }
     } else {
       callAPI(dispatch, formData);
     }
   };
 }
+
+export function getDismissedHCANotification() {
+  return dispatch => {
+    dispatch({ type: FETCH_DISMISSED_HCA_NOTIFICATION_STARTED });
+    const url = `/notifications/dismissed_statuses/form_10_10ez`;
+
+    apiRequest(
+      url,
+      null,
+      data =>
+        dispatch({ type: FETCH_DISMISSED_HCA_NOTIFICATION_SUCCEEDED, data }),
+      ({ errors }) =>
+        dispatch({ type: FETCH_DISMISSED_HCA_NOTIFICATION_FAILED, errors }),
+    );
+  };
+}
+
+export function setDismissedHCANotification(status, statusEffectiveAt, update) {
```

Comment:
This isn't used yet?

Reply:
nope, I already have the follow up PR that does use it, though. I broke my branch into two different commits to keep the PRs a little easier to review.